vms/evm/vmerrors/errors.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package vmerrors
+
+import "errors"
+
+var (
+	ErrGenerateBlockFailed         = errors.New("failed to generate block")
+	ErrBlockVerificationFailed     = errors.New("failed to verify block")
+	ErrWrapBlockFailed             = errors.New("failed to wrap block")
+	ErrInvalidCoinbase             = errors.New("invalid coinbase")
```

Comment:
This is only used for a unit test and it's defined by `dummy`. It seems we have an integration test [here](//github.com/ava-labs/coreth) that does an assertion on it, but I would argue that:
1. The test looks significantly more complicated than it needs to be, I think the minimal case you need is to build a block and overwrite the coinbase and it gives us this error (i'm not sure though)
2. Since this is defined by `dummy` and you don't have an import issue, we could just define this in `dummy` (since it defines this error anyways) and do an assert against `dummy.ErrInvalidCoinbase`